src/core/lib/transport/interception_chain.h

```diff
@@ -92,28 +79,43 @@ inline auto HijackCall(UnstartedCallHandler unstarted_call_handler,
 //
 // 2. It can consume the call by calling `Consume`.
 //
-// Upon the StartCall call the UnstartedCallHandler will be from the last
```

Comment:
Isn't this comment still relevant?  I think it's still important to explain the implications of using "unprocessed initial metadata" here.

Reply:
yeah it is... i got excited when i started writing this that the problem was solved (it's actually not - but maybe tractable)